contrib/src/test/java/com/datatorrent/contrib/kafka/KafkaInputOperatorTest.java

```diff
@@ -234,7 +236,20 @@ public void testKafkaInputOperator_Invalid() throws Exception
   public void afterTest()
   {
     tupleCount.set(0);
```

Comment:
Why is this done after the test? It should done before as you cannot guarantee how previous runs were terminated.